tango_simlib/tango_sim_generator.py

```diff
@@ -248,7 +263,7 @@ def _reset_to_default_state(self):
                 adjustable_attrs = simulated_quantity.adjustable_attributes
                 for attr in adjustable_attrs:
                     if attr == 'last_update_time':
-                        simulated_quantity.attr = model_time
+                        setattr(simulated_quantity, attr, model_time)
```

Comment:
In this case there is only one attribute, so the generic `setattr` mechanism isn't that useful.  At least the error of writing to the wrong attribute has been fixed!